nova/conf/base.py

```diff
@@ -195,9 +195,19 @@
 """),
 ]
 
+metrics_opts = [
+    cfg.IntOpt('statsd_port',
+               default=8125,
+               help='Port of the statsd service.'),
```

Comment:
We already have a `statsd_port` in our `oslo.messaging` [patches](https://github.com/sapcc/oslo.messaging/blob/66a77c535c089272accb87e871ade45474f314cf/oslo_messaging/server.py#L59-L69). Is it possible to register an option twice at the same place (`[Default]`)?